ups/decam_hits.table

```diff
@@ -0,0 +1,16 @@
+# For each dependency except python, list it here along with its minimum
+# version number.
+# Very common third-party packages (boost, python, swig, doxygen) 
+# and very low-level LSST packages can be assumed to be recursively
+# included by low-level LSST packages such as utils or daf_base.
+# Any other package whose interface is used should be listed explicitly
+# rather than assuming it will be included recursively.
+setupRequired(obs_decam)
+
+# The following is boilerplate for all packages.
+# See Tech Note DMTN-001 for details on LSST_LIBRARY_PATH
+# Pure-Python packages only need the last line.
+envPrepend(LD_LIBRARY_PATH, ${PRODUCT_DIR}/lib)
+envPrepend(DYLD_LIBRARY_PATH, ${PRODUCT_DIR}/lib)
+envPrepend(LSST_LIBRARY_PATH, ${PRODUCT_DIR}/lib)
+envPrepend(PYTHONPATH, ${PRODUCT_DIR}/python)
```

Comment:
These `envPrepend` commands seem pointless -- there is no `lib` directory, and in a Python-only package there shouldn't be.